library/src/main/java/com/novoda/downloadmanager/LiteDownloadMigrationService.java

```diff
@@ -110,6 +137,7 @@ MigrationDownloadServiceBinder withCallback(Migrator.Callback migrationCallback)
 
         void bind() {
             LiteDownloadMigrationService.this.migrationCallback.onUpdate(LiteDownloadMigrationService.this.updateMessage);
```

Comment:
Do you want to align this statics with the way that is done in the download manager service?

Reply:
I think I have also done this in the follow-up branch 😄 I will ensure that this `Service` behaves, as much as possible, like the other Service in the library 😄